src/generator/AutoRest.Go/Model/CodeModelGo.cs

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using AutoRest.Core;
+using AutoRest.Core.Model;
+using AutoRest.Core.Utilities;
+
+namespace AutoRest.Go.Model
+{
+    public class CodeModelGo : CodeModel
+    {
+        public CodeModelGo()
+        {
+            NextMethodUndefined = new List<IModelType>();
+            PagedTypes = new Dictionary<IModelType, string>();
+        }
+
+        public string[] Version => CodeNamerGo.SDKVersionFromPackageVersion(
+                                            !string.IsNullOrEmpty(Settings.Instance.PackageVersion)
+                                                    ? Settings.Instance.PackageVersion
+                                                    : "0.0.0");
+
+        public override string Namespace
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(base.Namespace))
+                    return base.Namespace;
```

Comment:
Code Format: you have a number of if( .. ) statements that should have { } for the body. We don't like 

if( ... ) 
  something;

please use 

if( ... ) 
{ 
   something;
}